package.json

```diff
@@ -33,24 +33,24 @@
   "author": "Boxfish Studio <info@boxfish.studio>",
   "license": "Apache-2.0",
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^5.48.2",
-    "@typescript-eslint/parser": "^5.48.2",
+    "@typescript-eslint/eslint-plugin": "^5.49.0",
+    "@typescript-eslint/parser": "^5.49.0",
     "eslint": "^8.32.0",
-    "tsup": "^6.5.0",
-    "typescript": "^4.7.4"
+    "typescript": "^4.9.4",
+    "tsup": "^6.5.0"
   },
   "dependencies": {
     "@babel/core": "^7.20.12",
     "@babel/eslint-parser": "^7.19.1",
     "@metaplex/arweave-cost": "^1.0.4",
```

Comment:
I think we should also update `@metaplex/arweave-cost`
![image](https://user-images.githubusercontent.com/72078237/214266874-acade2a0-bafc-4a95-8362-3ea6517193dc.png)

Reply:
@evavirseda thing is i did it on purpose. Last time I tried 2.0.0 it had an error, in which the cost was lower than it should, so I could not upload anything to arweave at all :|

Reply:
that's great, then :ok_hand: